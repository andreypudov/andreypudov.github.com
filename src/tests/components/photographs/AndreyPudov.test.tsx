import { MemoryRouter } from 'react-router-dom'
import Component from './../../../components/photographs/AndreyPudov';
import { HelmetProvider } from 'react-helmet-async';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <HelmetProvider>
        <MemoryRouter>
          <Component />
        </MemoryRouter>
      </HelmetProvider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});