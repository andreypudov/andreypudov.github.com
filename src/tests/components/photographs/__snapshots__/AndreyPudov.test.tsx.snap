// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <header>
    <div
      className="container"
    >
      <div
        className="navbar navbar-expand-md navbar-dark"
      >
        <div
          className="container-fluid"
        >
          <a
            className="navbar-brand mx-auto"
            href="/en/"
            onClick={[Function]}
          >
            Header.AndreyPudov
          </a>
          <button
            aria-controls="navbarMain"
            aria-expanded="false"
            aria-label="Toggle navigation"
            className="navbar-toggler"
            data-bs-target="#navbarMain"
            data-bs-toggle="collapse"
            type="button"
          >
            <span
              className="navbar-toggler-icon"
            />
          </button>
          <div
            className="collapse navbar-collapse justify-content-sm-center"
            id="navbarMain"
          >
            <ul
              className="navbar-nav"
            >
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/albums/"
                  onClick={[Function]}
                >
                  Albums
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/contents/"
                  onClick={[Function]}
                >
                  Contents
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/resume/"
                  onClick={[Function]}
                >
                  Resume
                </a>
              </li>
              <li
                className="nav-item"
              >
                <a
                  className="nav-link"
                  href="/en/contact/"
                  onClick={[Function]}
                >
                  Contact
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </header>
  <main>
    <div
      className="container"
    >
      <h1>
        Andrey Pudov
        <small
          className="text-muted"
        >
          Saturday, July 9, 2022
        </small>
      </h1>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_001.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_004.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_007.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_009.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_010.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_011.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_012.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_017.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_022.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_023.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_025.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_026.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_033.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2022-07-09/2022-07-09_034.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-09/2022-07-09_037.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2022-07-04/2022-07-04_001.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2016-04-16/2016-04-16_002.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image vertical"
          src="/photographs/2016-04-16/2016-04-16_004.webp"
          title="Andrey Pudov"
        />
      </figure>
      <figure>
        <img
          alt="Andrey Pudov"
          className="img-responsive gallery-image"
          src="/photographs/2016-04-16/2016-04-16_008.webp"
          title="Andrey Pudov"
        />
      </figure>
    </div>
  </main>
  <footer>
    <div
      className="container"
    >
      <hr
        className="footer-divider"
      />
      <ul
        className="piped-links float-end"
      >
        <li
          className="first piped-links-short"
        >
          <a
            href="/en/resume/"
            onClick={[Function]}
          >
            About
          </a>
        </li>
        <li
          className="first piped-links-long"
        >
          <a
            href="/en/resume/"
            onClick={[Function]}
          >
            About Andrey Pudov
          </a>
        </li>
        <li>
          <a
            href="/en/contact/"
            onClick={[Function]}
          >
            Contact
          </a>
        </li>
      </ul>
      <ul
        className="piped-links"
      >
        <p
          className="piped-links-long"
        >
          Copyright © 2023 Andrey Pudov. All rights reserved.
        </p>
        <p
          className="piped-links-short"
        >
          © 2023 Andrey Pudov. All rights reserved.
        </p>
      </ul>
    </div>
  </footer>
</div>
`;
